utils.js

```diff
@@ -39,3 +39,6 @@ export function getTemplateRefs(element) {
     }
   })
 }
+
+window.getTemplateRefs = getTemplateRefs;
+window.bindTemplateEvents = bindTemplateEvents;
```

Comment:
So these aren't working because the intention is to import them where needed (in the web component file), but I haven't got around to testing how to do that in a sfwc yet. Will merge this for now, but hopefully they import as expected in an actual sfwc (and hopefully a build tool can figure it out too, something else I'm planning on trying)